standard/requirements/edr/REQ_rc-point-coords-response.adoc

```diff
@@ -0,0 +1,30 @@
+[[req_edr_point-coords-response]]
+==== *Requirement /req/edr/point-coords-response* Parameter coords response
+
+[requirement,type="general",id="/req/edr/coords-response", label="/req/edr/point-coords-response"]
+====
+
+*A:*
+
+Only those resources that are representative of the requested geometry that is defined by the `coords` parameter SHALL be part of the result set.
+
+---
+*B:*
+
+The coordinates SHALL consist of a Well Known Text (WKT) POINT or (if supported by the collection) MULTIPOINT geometry string.
+
+---
+*C:*
+
+If an unsupported  Well Known Text (WKT) geometry is requested a `405` error SHOULD be returned.
```

Comment:
I believe that HTTP 405 is reserved strictly for HTTP method (GET, HEAD, POST, PUT, ...) mismatch, i.e. client used an HTTP method which is not supported by the endpoint.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405

Reply:
In PR#390